tests/e2e/tests/_legacy-experience/checkout/saved-card.spec.js

```diff
@@ -13,7 +14,7 @@ let username, userEmail;
 
 test.beforeAll( async () => {
 	// This allow multiple tests to run in parallel.
-	const randomString = Date.now();
+	const randomString = randomUUID();
```

Comment:
I sometimes got "User already exists" errors when running tests locally, so I figured using a random UUID would be best.